packages/gitgraph-react/src/stories/3-events.stories.tsx

```diff
@@ -1,15 +1,14 @@
 import * as React from "react";
 import { storiesOf } from "@storybook/react";
+import { action } from "@storybook/addon-actions";
 
-import { Gitgraph, CommitOptions } from "../Gitgraph";
+import { Gitgraph } from "../Gitgraph";
 
 storiesOf("3. Events", module)
   .add("on commit dot click", () => (
     <Gitgraph>
       {(gitgraph) => {
-        const onClick: CommitOptions["onClick"] = (commit) => {
-          alert(`Commit ${commit.hashAbbrev} clicked: "${commit.subject}"`);
-        };
+        const onClick = action("click on dot");
```

Comment:
I realized we didn't used `action` addon, and this was the perfect use case. Way better than the `alert`!